packages/@aws-cdk/aws-lambda/lib/function.ts

```diff
@@ -145,15 +145,30 @@ export interface FunctionProps {
 
   /**
    * What security group to associate with the Lambda's network interfaces.
+   * This property is being deprecated, consider using securityGroups instead.
    *
    * Only used if 'vpc' is supplied.
+   * Only use this or the securityGroups prop, not both at once
    *
    * @default - If the function is placed within a VPC and a security group is
-   * not specified, a dedicated security group will be created for this
-   * function.
+   * not specified, either by this or securityGroups prop, a dedicated security
+   * group will be created for this function.
    */
+  @deprecated
   readonly securityGroup?: ec2.ISecurityGroup;
 
+  /**
+   * What list of security groups to associate with the Lambda's network interfaces.
```

Comment:
language: `The list of security groups...`

Reply:
Sure, I was following what's already used for `securityGroup` prop.